src/main/webapp/modal-activator.js

```diff
@@ -35,4 +35,34 @@ function closeModal(modal) {
   if (modal == null) return;
   overlay.classList.remove('active');
   modal.classList.remove('active');
+}
+
+function populateResults(placesArray) {
+  console.log('Populating results modal...');
+  const listContainer = document.getElementById('results-list');
+  const entireList = document.createElement('ul');
+  places.forEach(place, () => {
+    entireList.appendChild(generateResult(place));
```

Comment:
generateResult() has no return statement, not sure if this is intended

Reply:
Addressed.